dotcom-rendering/src/components/QuoteIcon.tsx

```diff
@@ -17,7 +17,7 @@ type Props = {
  */
 export const QuoteIcon = ({ colour }: Props) => (
 	/* This viewBox is narrower than Source’s SvgQuote */
-	<svg viewBox="4 4 24 16" css={quoteStyles(colour)}>
-		<path d="M9.2776 8H14.0473C13.4732 12.5489 12.9653 17.0095 12.7445 22H4C4.79495 17.142 6.4511 12.5489 9.2776 8ZM20.3852 8H25.0887C24.5808 12.5489 24.0067 17.0095 23.7859 22H15.0635C15.9688 17.142 17.5587 12.5489 20.3852 8Z" />
+	<svg viewBox="0 0 22 14" css={quoteStyles(colour)}>
```

Comment:
`quoteStyles` should no longer need to be a function.

Reply:
Not sure I understand why!

Reply:
I thought that it would always take the `currentColor` of the surrounding text, but that’s not correct, so this is fine!

You might want to use the `style` property still, [as per Emotion Best Practices](https://emotion.sh/docs/best-practices#use-the-style-prop-for-dynamic-styles), which would then remove the need for `quoteStyles` to be a function.

```tsx
const base = css`/* stuff */`

const QuoteIcon = ({ colour }: Props) =>
  <svg css={base} style={{ color: colour }}><path d="M0 0Z" /></svg>
```

Reply:
Thanks for the suggestion! I'll make these changes

Reply:
fyi That didn't work as expected so reverted the [commit](https://github.com/guardian/dotcom-rendering/pull/9312/commits/987c1ed848cfc024ac0ad54bf1e78c75bf468262) and will revisit after I merge this.